chapters/en/chapter13/3.mdx

```diff
@@ -0,0 +1,324 @@
+# Understanding the DeepSeek R1 Paper
+
+This chapter is a crash course paper reading. We will walk through the paper in simple terms, and then we will break down the key concepts and takeaways.
+
+DeepSeek R1 represents a significant advancement in language model training, particularly in developing reasoning capabilities through reinforcement learning. The paper introduces a new reinforcement learning algorithm called Generalized Policy Rectification Optimization (GRPO).
```

Comment:
technically, it's not DeepSeek R1 that introduced GRPO. It's DeepSeek Math https://arxiv.org/abs/2402.03300